ci: build failing due to Black formatting errors

## 🐛 Bug Description
The latest merge to `develop` caused the CI pipeline to fail.
The `Check format with Black` step failed because files were not formatted correctly before merging.

## ❌ Actual Behavior
```text
would reformat src/data_analyzer/cleaner.py
would reformat tests/test_cleaner.py
Error: Process completed with exit code 1.
```

## ✅ Expected Behavior
CI should pass with green checks.

--- a/src/data_analyzer/cleaner.py
+++ b/src/data_analyzer/cleaner.py
@@ -32,7 +32,9 @@ class DataCleaner:
         except FileNotFoundError:
             raise FileNotFoundError(f"File not found: {filepath}")
 
-    def clean_data(self, drop_na_cols: Optional[List[str]] = None) -> pd.DataFrame:  # noqa: E501
+    def clean_data(
+        self, drop_na_cols: Optional[List[str]] = None
+    ) -> pd.DataFrame:  # noqa: E501
         """
         :param drop_na_cols: List of column names to check for NaNs.
                             If None, drops rows where ANY column is NaN.

--- a/src/data_analyzer/cleaner.py
+++ b/src/data_analyzer/cleaner.py
@@ -32,11 +32,11 @@ class DataCleaner:
         except FileNotFoundError:
             raise FileNotFoundError(f"File not found: {filepath}")
 
-    def clean_data(self, drop_na_cols: Optional[List[str]] = None) -> pd.DataFrame:
+    def clean_data(self, drop_na_cols: Optional[List[str]] = None) -> pd.DataFrame:  # noqa: E501
         """
-        :param drop_na_cols: List of column names to check for NaNs. 
+        :param drop_na_cols: List of column names to check for NaNs.
                             If None, drops rows where ANY column is NaN.
-        
+
         Perform standard cleaning operations:
         1. Normalize column names (lowercase, spaces to underscores)
         2. Remove duplicate rows
@@ -63,8 +63,9 @@ class DataCleaner:
 
         final_rows = len(self.df)
         # 替换 print 为 logging
-        logger.info(f"Cleaned data: Removed {initial_rows - dedup_rows} duplicates, "
-                    f"{dedup_rows - final_rows} rows with missing values.")
-        
+        logger.info(
+            f"Cleaned data: Removed {initial_rows - dedup_rows} duplicates, "
+            f"{dedup_rows - final_rows} rows with missing values."
+        )
 
         return self.df

--- a/tests/test_cleaner.py
+++ b/tests/test_cleaner.py
@@ -61,13 +61,15 @@ def test_clean_data_with_subset(messy_data):
     """Test cleaning with specific columns for dropna."""
     # Row 4: NaN Name, 40 Age, Chicago City
     # Row 5: Charlie Name, 35 Age, None City
-    
+
     cleaner = DataCleaner(messy_data)
-    
+
     # 只根据 'Name ' (normalized to 'name') 去除空值
     # Row 4 (NaN Name) 应该被删掉
     # Row 5 (None City) 应该保留
-    cleaned_df = cleaner.clean_data(drop_na_cols=['name'])
-    
-    assert len(cleaned_df) == 3  # Alice, Bob, Charlie (Bob is OK, Row 3 duplicate dropped)
-    assert 'Charlie' in cleaned_df['name'].values
+    cleaned_df = cleaner.clean_data(drop_na_cols=["name"])
+
+    assert (
+        len(cleaned_df) == 3
+    )  # Alice, Bob, Charlie (Bob is OK, Row 3 duplicate dropped)
+    assert "Charlie" in cleaned_df["name"].values

